crates/nu-std/std/formats.nu

```diff
@@ -9,12 +9,22 @@
 # These functions help `open` the files with unsupported extensions such as ndjson.
 #
 
-# Convert from ndjson to structured data.
+# Convert from NDJSON to structured data.
```

Comment:
Should the help be more explicit?
In theory we could point to the "spec" http://ndjson.org/

```suggestion
# Convert from newline-delimited JSON to structured data.
```

Reply:
i went with a259805d0, is that ok? :innocent: